test/compilable/testcolor.sh

```diff
@@ -0,0 +1,46 @@
+set -eo pipefail
```

Comment:
`pipefail` is a bash feature, please add a bash shebang as in the other scripts.